### PR TITLE
[MOBILE-648b] Release 8.0.0

### PR DESCRIPTION
**Pull request by crow**
### What do these changes do?
Updates urbanairship-cordova ios and android platforms to use their latest respective airship SDKs.

- Runs the update_cordova script for both platforms
- Makes necessary changes to migrate each plugin to newest platform

### Why are these changes necessary?
To get both Cordova platforms to use the latest version of the airship SDK and to lay the groundwork for further changes - such as support for loading custom notification channels.

### How did you verify these changes?
Manually tested iOS and Android samples. 

- Tracked execution to appropriate location delegate callbacks in iOS sample.
- Sent/opened landing pages, message center on android sample.

#### Verification Screenshots:
landing_page_cordova
message_center
message_view

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,11 @@
+=============================
+Version 8.0.0 - July 10, 2019
+=============================
+- Updated Android SDK to 10.0.1
+- Updated iOS SDK to 11.1.0
+- Added support for creating new Android notification channels and setting the default notification channel ID in Android
+- Using Airship location in iOS now requires linking AirshipLocationKit
```
Comment (rlepinski) [inline CHANGELOG]: new plugin

File: src/android/UAirshipPlugin.java [hunk 2]
```diff
@@ -85,6 +89,11 @@
     private final static List<String> GLOBAL_ACTIONS = Arrays.asList("takeOff", "registerListener", "setAndroidNotificationConfig");
     private final ExecutorService executorService = Executors.newFixedThreadPool(1);
 
+    private static final String NOTIFICATION_ICON_KEY = "icon";
+    private static final String NOTIFICATION_LARGE_ICON_KEY = "largeIcon";
+    private static final String ACCENT_COLOR_KEY = "accentColor";
+    private static final String DEFAULT_CHANNEL_ID_KEY = "defaultChannelId";
+
```
Comment (rlepinski) [inline src/android/UAirshipPlugin.java]: Need to remove the location calls from this plugin

File: src/ios/UAirshipPlugin.m [hunk 3]
```diff
@@ -879,5 +902,39 @@ - (BOOL)notifyListener:(NSString *)eventType data:(NSDictionary *)data {
...
+- (void)closeOverlayMessage {
+    UA_LTRACE(@"closeOverlayMessage called");
+
+    UIViewController *vc = [self.htmlAdapter valueForKey:@"htmlViewController"];
+# pragma clang diagnostic push
+# pragma clang diagnostic ignored "-Warc-performSelector-leaks"
+    [vc performSelector:NSSelectorFromString(@"dismissWithoutResolution")];
+# pragma clang diagnostic pop
+}
+
```
Comment (rlepinski) [inline src/ios/UAirshipPlugin.m]: Need to remove the location calls

File: www/UrbanAirship.js [hunk 4]
```diff
@@ -606,6 +618,10 @@ module.exports = {
   /**
    * Checks if background location is enabled or not.
    *
+   * Note: On iOS, location services require an additional dependency
+   * on AirshipLocationKit. For more information see
+   * https://docs.airship.com/platform/cordova/location/
```
Comment (rlepinski) [inline www/UrbanAirship.js]: Need to remove the location calls

File: src/ios/UACordovaPluginManager.h [hunk 5]
```diff
@@ -1,7 +1,14 @@
 /* Copyright Urban Airship and Contributors */
 
 #import <Foundation/Foundation.h>
+
+#if __has_include(<AirshipKit/AirshipLib.h>)
```
Comment (marc-scig) [inline src/ios/UACordovaPluginManager.h]: is this dance necessary outside of UALocation?

Comment (crow) [reply src/ios/UACordovaPluginManager.h]: if the cocoapod tags are used the cordova system will choke depending on if use_frameworks! flag is present, this protects from build issue while using said cocoapod tags

File: README.md [hunk 6]
```diff
@@ -18,8 +18,25 @@ One of our engineers will verify receipt of the agreement before approving your
...
+Please run the command `pod repo update` and re-add the plugin to resolve this issue.
+
+You would only run `pod repo update` if you have the specs-repo already cloned on your machine through `pod setup`.
+
 ### Requirements:
- - Cordova-CLI >= 7.0.0
+ - Cordova-CLI >= 9.0.0
 
 #### iOS:
 - Xcode 10+
```
Comment (rlepinski) [inline README.md]: Cocoapods

Comment (rlepinski) [reply README.md]: cordova-ios >= 5.0.1

File: CHANGELOG [hunk 7]
```diff
@@ -1,3 +1,11 @@
+=============================
+Version 8.0.0 - July 10, 2019
+=============================
+- Updated Android SDK to 10.0.1
```
Comment (rlepinski) [inline CHANGELOG]: Mention the new cordova requirements

**Pull request merged**